data-es6.js

```diff
@@ -18386,7 +18386,7 @@ exports.tests = [
 
         a >= 0;
         b in {};
-        c === 0;
+        c == 0;
```

Comment:
ah, in this case the == coercion was actually the point of the test :-/
```suggestion
        c == 0; // eslint-disable-line eqeqeq
```